### PR TITLE
Helm chart refresh

### PR DESCRIPTION
**Pull request by invertigo**
## 📚 Purpose
Update helm chart

## 📦 Impacts:
`helm/mds`

## ✅ PR Checklist
Build images from PR 616 `pnpm image:development` and then try `helm install mds ./helm/mds --values ./helm/mds/values-minimal.yaml` on a machine running docker desktop

### CODE REVIEW

File: helm/mds/requirements.lock [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  version: 4.5.1
+  version: 6.0.0
 - name: kafka
   repository: https://charts.bitnami.com/bitnami
-  version: 11.5.1
-digest: sha256:8d0e7ad1a7365385bc25fb7a22dc794e799507383ff0c71fd177495e0482ef19
-generated: "2020-12-23T19:16:36.701964-05:00"
+  version: 12.2.2
+digest: sha256:5def2abc8234bc9028c91439b240c3e25c89f75cc645cc3b529d506b1cd11f9a
+generated: "2020-12-03T09:25:34.200729567-08:00"
```
Comment (janedotx) [inline helm/mds/requirements.lock:15]: Is this timestamp really correct? It looks like we're going back in time. Also shouldn't the year be 2021?

Comment (avatarneil) [reply helm/mds/requirements.lock:15]: This is typically auto-generated (original version likely came around when one of us was trying to get local installs working again in the winter), but I think there are just some updates in here copy-pasted from an external chart (hence the old generated date). I don't think we're _really_ going back in time, but I wonder if having the `generated` and `digest` tags are even necessary (given some of the entries are lacking them)

**Review by avatarneil — COMMENTED** [review]
Purge any `lacuna` pls

File: helm/mds/requirements.lock (hunk 1 above)
Comment (janedotx) [reply helm/mds/requirements.lock:15]: I mean, I know we're not going back in time, but this timestamp seems kinda misleading.

**Pull request closed without merging**